projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.html

```diff
@@ -46,6 +46,9 @@
                     [height]="pdfViewer.offsetHeight"
                     [pageHeights]="pageHeights">
     </mv-comment-set>
+    <mv-bookmark [zoom]="zoom"
```

Comment:
maybe rename this to reflect that it's the bookmark icon/placeholder, rather than the actual bookmark